tools/src/test/java/org/apache/kafka/tools/DelegationTokenCommandTest.java

```diff
@@ -81,6 +92,27 @@ public void testDelegationTokenRequests() throws ExecutionException, Interrupted
 
     }
 
+    @ClusterTest(types = { Type.KRAFT },
+        brokerSecurityProtocol = SecurityProtocol.SASL_PLAINTEXT,
+        controllerSecurityProtocol = SecurityProtocol.SASL_PLAINTEXT,
+        serverProperties = {
+            @ClusterConfigProperty(key = DELEGATION_TOKEN_SECRET_KEY_CONFIG, value = "key")
+        })
+    public void testDescribeDelegationTokenWithBootstrapController(ClusterInstance clusterInstance) throws ExecutionException, InterruptedException {
```

Comment:
This test case is about Admin. We need another test case to test DelegationTokenCommand directly.